sass/tji/_filters.scss

```diff
@@ -0,0 +1,31 @@
+/*--------------------------------------------------------------
+## FILTERS
+--------------------------------------------------------------*/
+.tji-chartview-filters {}
+
+.tji-chartview-filters__filter-legend {
+  svg {
+    @include transition(transform, .5s);
+  }
+  &.is-collapsed {
+    svg {
+      @include transform(rotate(180deg));
+    }
+  }
+}
+
+.tji-chartview-filters__filter-set {
+  @include transition(max-height, .5s);
+  max-height: 500px;
+  overflow-y: hidden;
```

Comment:
Why hidden instead of scroll? Won't this prevent users from seeing some of their filter contents in some cases?

Reply:
Setting a max-height and overflow-y hidden is a [technique](https://css-tricks.com/using-css-transitions-auto-dimensions/) used to allow for collapse/expand animations. It's kind of not ideal since it means our filter lists is constrained to less than 500px. But honestly, if we have checkboxes that extend beyond 500px, we need to rethink our UI for displaying all these filters.